src/scripts/cervical_cancer_analyses/cervical_cancer_analyses.py

```diff
@@ -0,0 +1,602 @@
+* Check key outputs for reporting in the calibration table of the write-up
+* Produce representative plots for the default parameters
+
+NB. To see larger effects
+* Increase incidence of cancer (see tests)
+* Increase symptom onset
+* Increase progression rates (see tests)
+"""
+import datetime
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from tlo import Date, Simulation, logging
+from tlo.analysis.utils import parse_log_file
+from tlo.methods import (
+    cervical_cancer,
+    demography,
+    enhanced_lifestyle,
+    epi,
+    healthburden,
+    healthseekingbehaviour,
+    healthsystem,
+    hiv,
+    simplified_births,
+    symptommanager,
+    tb,
+)
+
+seed = 12
+
+# Date-stamp to label log files and any other outputs
+datestamp = datetime.date.today().strftime("__%Y_%m_%d")
+
+# The resource files
+resourcefilepath = Path("./resources")
+
+log_config = {
+    "filename": "cervical_cancer_analysis",   # The name of the output file (a timestamp will be appended).
+    "directory": "./outputs",  # The default output path is `./outputs`. Change it here, if necessary
+    "custom_levels": {  # Customise the output of specific loggers. They are applied in order:
+        "*": logging.WARNING,  # Asterisk matches all loggers - we set the default level to WARNING
+        "tlo.methods.cervical_cancer": logging.INFO,
+        "tlo.methods.healthsystem": logging.INFO,
+    }
+}
+
+# Set parameters for the simulation
+start_date = Date(2010, 1, 1)
+end_date = Date(2013, 1, 1)
+malawi_country_pop = 17000000
+popsize = 170000
+
+def run_sim(service_availability):
+    # Establish the simulation object and set the seed
+    sim = Simulation(start_date=start_date, seed=seed, log_config=log_config)
+#     sim = Simulation(start_date=start_date, log_config={"filename": "logfile"})
+
+    # Register the appropriate modules
+    sim.register(demography.Demography(resourcefilepath=resourcefilepath),
+                 cervical_cancer.CervicalCancer(resourcefilepath=resourcefilepath),
+#                cc_test.CervicalCancer(resourcefilepath=resourcefilepath),
+                 simplified_births.SimplifiedBirths(resourcefilepath=resourcefilepath),
+                 enhanced_lifestyle.Lifestyle(resourcefilepath=resourcefilepath),
+                 healthsystem.HealthSystem(resourcefilepath=resourcefilepath,
+                                           disable=False,
+                                           cons_availability='all'),
+                 symptommanager.SymptomManager(resourcefilepath=resourcefilepath),
+                 healthseekingbehaviour.HealthSeekingBehaviour(resourcefilepath=resourcefilepath),
+                 healthburden.HealthBurden(resourcefilepath=resourcefilepath),
+                 epi.Epi(resourcefilepath=resourcefilepath),
+                 tb.Tb(resourcefilepath=resourcefilepath, run_with_checks=False),
+                 hiv.Hiv(resourcefilepath=resourcefilepath, run_with_checks=False)
```

Comment:
@andrew-phillips-1  and @mmsuarezcosta  - note that you could be using the dummy HIV module for +++speed